cucumber/api/features/support/conjur_token.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class ConjurToken
+  attr_reader :token
+
+  def initialize(raw_token)
+    @token = JSON.parse(raw_token)
+  end
+
+  def username
+    payload['sub']
+  end
+
+  private
+
+  def payload
+    @payload ||= JSON.parse(Base64.decode64(@token['payload']))
+  end
+end
```

Comment:
This is a general comment, not anything you need to change:

This is a nice abstraction, and probably belongs in the code itself, and not merely as a test helper.  I imagine (I haven't looked though) that this `Base64` logic is spread out in multiple places across the code base, a perfect example of [connascense of algorithm](https://practicingruby.com/articles/connascence) code smell.

The fix would be to have this object (or one like it) which is used everywhere.

Reply:
@liavyona @jonahx Regarding...

> The fix would be to have this object (or one like it) which is used everywhere.

We do have this token class that we use to create the token: https://github.com/cyberark/slosilo/blob/master/lib/slosilo/jwt.rb

Could this be reused here? Example:
```
Slosilo::JWT.parse_json(raw_token)
```

Reply:
@micahlee  It works great. Thanks.